### PR TITLE
summarize_graph show framework error when input saved_model is selected

### PR DESCRIPTION
**Pull request by vgavrilo**
Root cause analysis: For some TF saved_model, we have a framework error after running the script `summarize_graph.py`.

> mo.utils.error.FrameworkError: SavedModel format load failure: MetaGraphDef associated with tags '' could not be found in SavedModel. To inspect available tag-sets in the SavedModel, please use the SavedModel CLI: `saved_model_cli`
available_tags: [\{'serve'}].

The reason of this error is the following: the value of the command line key `--saved_model_tags` must be considered as a group of tag(s) of the MetaGraphDef to load, in string format, separated by commas. Hence, this value must be split into a list of strings by commas. But such splitting isn't performed.

Solution: add splitting the value of the command line key `--saved_model_tags` by commas.

### Tickets:
 - *58810*

Code:
* [x]  Comments
* [x]  Code style (PEP8)
* [x]  Transformation generates reshape-able IR: N/A because there are no new transformations
* [x]  Transformation preserves original framework node names: N/A because there are no new transformations

Validation:
* [x]  Unit tests: N/A
* [x]  Framework operation tests: N/A - No new enabled operations;
* [x]  Transformation tests: N/A - No new implemented transformations;
* [x]  Model Optimizer IR Reader check: N/A

Documentation:
* [x]  Supported frameworks operations list: N/A - No new supported operations;
* [x]  Supported **public** models list: N/A - Nothing to add;
* [x]  User guide update: N/A - Nothing to update.

### CODE REVIEW

**Comment by vgavrilo** [comment]
@iimironov, @rkazants, @popovaan, @lazarevevgeny, please, see this PR (#6726)

File: model-optimizer/mo/utils/summarize_graph.py [hunk 1]
```diff
@@ -70,9 +70,10 @@ def summarize_graph(graph_def):
     if argv.input_model and argv.saved_model_dir:
         print("[ ERROR ] Both keys were provided --input_model and --input_dir. Please, provide only one of them")
         sys.exit(1)
-    graph_def, _ = load_tf_graph_def(graph_file_name=argv.input_model, is_binary=not argv.text,
-                                     checkpoint=argv.input_checkpoint,
-                                     model_dir=argv.saved_model_dir, saved_model_tags=argv.saved_model_tags)
+    tags = argv.saved_model_tags.split(",")
```
Comment (iimironov) [inline model-optimizer/mo/utils/summarize_graph.py:73]: Please add check if argv.saved_model_tags not None

Comment (vgavrilo) [reply model-optimizer/mo/utils/summarize_graph.py:73]: This key has a default value, and this default value is an empty string (see lines 61-63 of the file `summarize_graph.py`).

**Comment by vgavrilo** [comment]
@iimironov, @rkazants, @popovaan, @lazarevevgeny, please, see this PR (#6726)

File: model-optimizer/mo/utils/summarize_graph.py [hunk 2]
```diff
@@ -70,9 +70,10 @@ def summarize_graph(graph_def):
...
+    graph_def, _, _ = load_tf_graph_def(graph_file_name=argv.input_model, is_binary=not argv.text,
```
Comment (popovaan) [inline model-optimizer/mo/utils/summarize_graph.py:74]: Can we add some unit test for this case? May be extend TestingSummarizeGraph?

Comment (vgavrilo) [reply model-optimizer/mo/utils/summarize_graph.py:74]: It is hard to write: tags are used only for saved_model format, but this format requires a special structure directory for model, not a pb-file. And more, this fix does processing of the command line parameter `--saved_model_tags` according to the help for this parameter: the value of this parameter is split into a list of strings by commas.

**Comment by vgavrilo** [comment]
@iimironov, @rkazants, @popovaan, @lazarevevgeny, please, see this PR (#6726)

**Review by rkazants — APPROVED** [review]

**Comment by vgavrilo** [comment]
@iimironov, @popovaan, @lazarevevgeny, please, see this PR (#6726)

**Review by popovaan — APPROVED** [review]

**Pull request merged**